### PR TITLE
Matlab MonetRF implementation

### PR DESCRIPTION
**Pull request by olakiril**

### CODE REVIEW

**Comment by fabiansinz** [comment]
Why do we create double implementations of everything?  I don't think that's a good idea.

**Comment by olakiril** [comment]
In general I agree, but the current python implementation is terribly inefficient. For example I am processing the MonetRF of one site since yesterday, and while it is using 500GB of memory it has not finished yet. 
Old matlab code was miles better, so if anything lets switch to that until Dimitri finishes his latest version.

**Comment by fabiansinz** [comment]
Alright, but then raise an exception in the respective python code so that only one implementation is used at a time.

File: matlab/schemas/+tuning/MonetRF.m [hunk 1]
```diff
@@ -72,9 +72,11 @@ function makeTuples(self, key)
                     update = conv2(movie, snippet(:,itrace)', 'valid')';
                     update = interp1((0:size(update,1)-1)/fps, update, (0:nbins-1)*bin_width);  % resample 
                     update = reshape(update', sz(1), sz(2), nbins);
+                    if any(isnan(update(:)));fprintf('nan values for trial # %d, skipping...',trial.trial_idx);break;end
```
Comment (dimitri-yatsenko) [inline matlab/schemas/+tuning/MonetRF.m]: shouldn't this be `continue` rather than `break`?

**Pull request merged**